include/envoy/http/context.h

```diff
@@ -29,6 +34,7 @@ class Context {
 
   virtual CodeStats& codeStats() PURE;
   virtual const UserAgentContext& userAgentContext() const PURE;
+  virtual Stats::StatName asyncClientStatPrefix() const PURE;
```

Comment:
Maybe I'm missing something but how can this compile with only a forward declaration? Doesn't it need to be a pointer or reference?

Reply:
I changed this to return a const ref.